### DISCUSSION TITLE
Is barrier still being maintained?

### DISCUSSION THREAD

**Discussion by erickmoreno** [debauchee/barrier#2019] [Q&A]
I check this repo periodically looking for updates and fixes to my use (MacOs and Logitech mouse) but, even with some PRs open and discussions about the issue, nothing happens.

This project looks abandoned, does anyone know a bit more about it?

**Comment by bfelds5** [comment]
No but there's a fork. Answered here: https://github.com/debauchee/barrier/discussions/2012
